keyboards/und/pcm-t-k0-windows.xml

```diff
@@ -106,222 +108,26 @@
 		<map iso="B09" to=":"/>
 		<map iso="B10" to="&quot;"/>
 	</keyMap>
-	<keyMap modifiers="caps+shift">
```

Comment:
why are you disabling caps lock? Should be same as shift, eh?

Reply:
Added caps layer, with lower case top row and non-letter keys. This is how caps lock works on QWERTY.